db/system_keyspace.cc

```diff
@@ -2302,9 +2297,14 @@ future<std::set<sstring>> system_keyspace::load_local_enabled_features() {
     co_return features;
 }
 
-future<> system_keyspace::save_local_enabled_features(std::set<sstring> features) {
+future<> system_keyspace::save_local_enabled_features(std::set<sstring> features, bool durable) {
```

Comment:
What about SSTABLE_FORMAT_PARAM_NAME? Don't we need it early too, so that we don't write sstables in old format during cl replay?

Reply:
yes, fixed